source/org/zfin/sequence/DBLink.java

```diff
@@ -22,44 +24,69 @@
 
 @Setter
 @Getter
+@Entity
+@Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@DiscriminatorFormula(
+        "CASE get_obj_type(dblink_linked_recid) WHEN 'ORTHO' THEN 'ORTH' WHEN 'TERM' THEN 'TERM' " +
```

Comment:
Could you format the 'END' clauses each on a single line? That would improve readability.